docs/installation/_index.md

```diff
@@ -124,76 +175,113 @@ curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-serv
+<ol start="3">
+<li>Select the tab below matching your computer's architecture, and run the command listed to download and install <code>viam-server</code>.
+   We recommend the stable release for most users:
+
+{{< tabs name="linux-computer-architectures" >}}
+{{% tab name="aarch64" %}}
+
+**Stable:**
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo ./viam-server -config myconfig.json
+curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-stable-aarch64.AppImage -o viam-server &&
+  chmod 755 viam-server && sudo ./viam-server --aix-install
 ```
 
-#### Start automatically on boot
+**Latest:**
 
-If you install `viam-server` as a system service, `viam-server` starts automatically every time you boot your device.
-Note that this is the default way `viam-server` is set up if you follow the **Setup** tab instructions on the [Viam app](https://app.viam.com).
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-latest-aarch64.AppImage -o viam-server &&
+  chmod 755 viam-server && sudo ./viam-server --aix-install
+```
 
-For this setup your configuration file must be at <file>/etc/viam.json</file>.
+{{% /tab %}}
+{{% tab name="x86_64" %}}
 
-The following command creates a systemd service file at <file>/etc/systemd/system/viam-server.service</file> and sets it to start on boot, using a config placed at <file>/etc/viam.json</file>.
-It will also move the actual binary (AppImage) to <file>/usr/local/bin/viam-server</file> (regardless of the previous filename).
+**Stable:**
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo ./viam-server --aix-install
+curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-stable-x86_64.AppImage -o viam-server &&
+  chmod 755 viam-server && sudo ./viam-server --aix-install
 ```
 
-Start the service by running:
+**Latest:**
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo systemctl start viam-server
+curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-latest-x86_64.AppImage -o viam-server &&
+  chmod 755 viam-server && sudo ./viam-server --aix-install
 ```
 
-The service is an AppImage and will check for updates and self-update automatically each time the service is started.
-Self-updates can take a couple of minutes, so the service may sometimes take a moment to start while this runs.
-You can disable this by commenting out the ExecPre line (the one with --aix-update on it) in the service file.
+{{% /tab %}}
+{{< /tabs >}}
+
+</li>
+</ol>
+
+4. To make configuration changes to your robot, edit the <file>/etc/viam.json</file> configuration file, then stop and restart `viam-server`:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   sudo systemctl stop viam-server
+   sudo systemctl start viam-server
+   ```
```

Comment:
May also want to show `sudo systemctl disable viam-server` if they're on a computer where they don't want it starting automatically.